pkg/pillar/cmd/volumemgr/initialvolumestatus.go

```diff
@@ -117,6 +140,38 @@ func gcVolumes(ctx *volumemgrContext, locations []string) {
 	}
 }
 
+func getVolumeStatusByPVC(pvcName string) (*types.VolumeStatus, error) {
```

Comment:
How about adding a comment that this is the reverse of the PVCName function you added? That would be helpful for a reader.